packages/falter-berlin-tunnelmanager/files/up.sh

```diff
@@ -19,9 +19,81 @@ get_configured_ips() {
     ip -o -4 a s | awk -F ' |\/' '{print $7}'
 }
 
-get_ips_from_prefix "$prefix"
+cleanup_olsr_wg_config() {
+    # all_olsr_interfaces=$(echo "/int" | nc 127.0.0.1 2006 | tail +3 | awk '{print $1}')
```

Comment:
I would like to leave this comment to know how to get all interfaces. I can imagine a scenario in which we need this.

Reply:
Okay. So if this line can be useful for you, you should mention it in a comment, otherwise the line could get lost in a later refactoring step. Anyway I would be fine with either way.